mock-echo-app/src/cmr/mock_echo/client/echo_util.clj

```diff
@@ -181,27 +181,28 @@
         cmr-response (ac/create-acl context cmr-acl {:raw? true :token (config/echo-system-token)})
         ;;attempt to create ACL.  If it already exists, then get the existing ACL and update.
         cmr-response
-              (if (= 409 (:status cmr-response))
-                (let [existing-concept-id (->> (get-in cmr-response [:body :errors])
-                                               first
-                                               (re-find #"\[([\w\d-]+)\]")
-                                               second)
-                      existing-concept (:body (ac/get-acl context existing-concept-id {:raw? true :token (config/echo-system-token)}))
-                      updated-concept (assoc existing-concept
-                                             :group_permissions (into (:group_permissions existing-concept) group-permissions))]
-                  (assoc
-                   (ac/update-acl context existing-concept-id updated-concept {:raw? true :token (config/echo-system-token)})
-                   :acl updated-concept))
-                cmr-response)
+        (if (= 409 (:status cmr-response))
```

Comment:
This was done automatically by the editor. :) I normally don't format existing code in order to keep the code change footprint small.